GovUkDesignSystemComponents/Table.cshtml

```diff
@@ -0,0 +1,88 @@
+﻿@model GovUkDesignSystem.GovUkDesignSystemComponents.TableGovUkViewModel
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@{
+    string classNames =
+        $"govuk-table "
+        + $"{(Model == null || String.IsNullOrEmpty(Model.Classes) ? string.Empty : Model.Classes)}";
+
+    string captionClassNames =
+        $"govuk-table__caption "
+        + $"{(Model == null || String.IsNullOrEmpty(Model.CaptionClasses) ? string.Empty : Model.CaptionClasses)}";
+
+    List<TableCellViewModel> headerRow = new List<TableCellViewModel>();
+
+    if (Model.FirstCellIsHeader)
+    {
+        if (Model != null && Model.Rows != null && Model.Rows.Count > 0)
+        {
+            headerRow = Model.Rows[0].Row;
+        }
+    }
+    else
+    {
+        headerRow = Model.Head;
+    }
+}
+@if (Model != null)
```

Comment:
Do we need this check?

Reply:
Yes, I think so, otherwise a null model will result in just **<table></table>** tags.

Reply:
It's more that I think we just assume that Model is not null in a lot of places, so it seemed odd to check just here.

Reply:
Yeah that's right. Done, thanks